service/src/test/java/uk/nhs/adaptors/gp2gp/uat/EhrExtractUATTest.java

```diff
@@ -146,61 +105,65 @@ public void setUp() {
-        var hl7TranslatedResponse = outputMessageWrapperMapper.map(
-            getGpcStructuredTaskDefinition,
-            ehrExtractContent);
+        final EhrExtractTemplateParameters ehrExtractTemplateParameters =
+            ehrExtractMapper.mapBundleToEhrFhirExtractParams(getGpcStructuredTaskDefinition, bundle);
+
+        final String ehrExtractContent = ehrExtractMapper.mapEhrExtractToXml(ehrExtractTemplateParameters);
+
+        final String hl7TranslatedResponse = outputMessageWrapperMapper.map(getGpcStructuredTaskDefinition, ehrExtractContent);
 
         assertThat(hl7TranslatedResponse).isEqualTo(expectedJsonToXmlContent);
     }
 
     private static Stream<Arguments> testValueFilePaths() {
-        return Stream.of(
-            Arguments.of(INPUT_JSON_WITH_PATHOLOGY_RECORD, OUTPUT_XML_USES_PATHOLOGY_RECORD),
-            Arguments.of(INPUT_JSON_WITH_MEDICATION_RECORD, OUTPUT_XML_USES_MEDICATION_RECORD),
-            Arguments.of(INPUT_JSON_WITH_STRUCTURED_CONSULTATIONS_INCLUDING_NARRATIVE_CONTENT,
-                OUTPUT_XML_USES_STRUCTURED_CONSULTATIONS_INCLUDING_NARRATIVE_CONTENT),
-            Arguments.of(INPUT_JSON_WITH_GENERAL_MISCELLANEOUS_CODED_OBSERVATION_CONTENT,
-                OUTPUT_XML_USES_GENERAL_MISCELLANEOUS_CODED_OBSERVATION_CONTENT),
-            Arguments.of(INPUT_JSON_WITH_BLOOD_PRESSURES, OUTPUT_XML_USES_BLOOD_PRESSURES),
-            Arguments.of(INPUT_JSON_WITH_IMMUNISATIONS, OUTPUT_XML_USES_IMMUNISATIONS),
-            Arguments.of(INPUT_JSON_WITH_DRUG_AND_NON_DRUG_ALLERGIES, OUTPUT_XML_USES_DRUG_AND_NON_DRUG_ALLERGIES),
-            Arguments.of(INPUT_JSON_WITH_DIARY_ENTRIES, OUTPUT_XML_USES_DIARY_ENTRIES),
-            Arguments.of(INPUT_JSON_WITH_REFERRALS, OUTPUT_XML_USES_REFERRALS),
-            Arguments.of(INPUT_JSON_WITH_PROBLEMS_AND_PROBLEM_LINKAGES, OUTPUT_XML_USES_PROBLEMS_AND_PROBLEM_LINKAGES),
-            Arguments.of(INPUT_JSON_WITH_NON_SNOMED_CODED_DATA_VIA_EGTON_CODES, OUTPUT_XML_USES_NON_SNOMED_CODED_DATA_VIA_EGTON_CODES)
+        return Stream.of(Arguments.of("9465701483_Dougill_full_20210119.json", "9465701483_Dougill_full_20210119.xml"),
+            Arguments.of("9465701483_Nel_full_20210119.json", "9465701483_Nel_full_20210119.xml"),
+            Arguments.of("9465701459_Nel_full_20210119.json", "9465701459_Nel_full_20210119.xml"),
+            Arguments.of("9465698679_Gainsford_full_20210119.json", "9465698679_Gainsford_full_20210119.xml"),
+//            Arguments.of("9465700193_Birdi_full_20210119.json", "9465700193_Birdi_full_20210119.xml"),
+            Arguments.of("9465701262_Meyers_full_20210119.json", "9465701262_Meyers_full_20210119.xml"),
+            Arguments.of("9465699918_Magre_full_20210119.json", "9465699918_Magre_full_20210119.xml"),
+//            Arguments.of("9465701297_Livermore_full_20210119.json", "9465701297_Livermore_full_20210119.xml"),
+//            Arguments.of("9465700339_Yamura_full_20210119.json", "9465700339_Yamura_full_20210119.xml"),
```

Comment:
Commented out due to parsing exception:

```
[Fatal Error] :3154:113: The content of elements must consist of well-formed character data or markup.

The content of elements must consist of well-formed character data or markup.
org.xml.sax.SAXParseException; lineNumber: 3154; columnNumber: 113; The content of elements must consist of well-formed character data or markup.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:261)
```

XML:
`<referenceRange typeCode="REFV"><referenceInterpretationRange classCode="OBS" moodCode="EVN.CRT"><text>< 2.000 mm/h</text><value></value></referenceInterpretationRange></referenceRange>`

Reply:
Also failing to parse the Observation comment correctly when containing special characters and line breaks:

```
"comment": "Episode: First ever. NOTES: These are notes containing\n%^%!\"£$%^&*()-_+=';:#~/><.?/¬\nand some \nline\nbreaks."
```

Reply:
Issue raised for `referenceInterpretationRange` XML encoding error, [NIAD-1380](https://gpitbjss.atlassian.net/browse/NIAD-1380)